### ISSUE TITLE
Unable to Raid Colonies in The End using command, summon or eggs

### ISSUE THREAD

**Issue by smmmadden** [ldtteam/minecolonies#6782]
### Prerequisites 

- [ ] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [ x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x ] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 36.1.0
- MineColonies Version: 0.14.21-ALPHA
- Structurize Version: 01.13.148-ALPHA
- Related Mods and their Versions:

## Expected behavior

The dimension or world should not matter when t comes to colonies, citizens and raiding entities. But it does not spawn any of them.
## Actual behavior

Use the Span egg for the barbarian and chieftan and they appear briefly and disappear.  Using the command option results in no raids at all.

## Steps to reproduce the issue

1. 
2. 
3. 

### Logs

- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Raycoms** [comment]
Eggs and summon doesn't work because the raiders have to be associated to raiding. Do you have a colony in the end? Is the colony big enough to receive raids?

**Comment by smmmadden** [comment]
Yes I believe so.  It is almost done with all buildings about a half dozen decorations with 73 Citizens.
![image](https://user-images.githubusercontent.com/10309346/111869868-bcf0f480-8957-11eb-9f39-c0660b57f0cd.png)

**Comment by gattsuru** [comment]
Have you seen a boss style bar when you use the raid-now command?
![image](https://user-images.githubusercontent.com/39036909/111884025-3b1fbc00-8995-11eb-995c-d4fd2812ac6e.png)


They _should_ be able to spawn as long as you're under the maxYforBarbarians value from the server-config, which defaults to 200.  The majority will be barbarians, however, and the raid generator doesn't strictly enforce successful spawn locations.  Depending on where and how of your colony, there's a pretty good chance that they'll wander off cliffs or spawn just above y=0 (and then drop into the void/burn up on their campfire).

**Comment by Raycoms** [comment]
The end has a lot of "void" so there is a relatively high chance for a raid to not find any valid position. We have neither optimized raids for nether nor for the end or similar dimensions yet. Nether is planned, "void" dimensions are significantly harder.
I will move this to be a "feature request" and rename the title to fit it properly.
